src/v2/Apps/Artist2/Components/Artist2Meta.tsx

```diff
@@ -0,0 +1,319 @@
+import { Artist2Meta_artist } from "v2/__generated__/Artist2Meta_artist.graphql"
+import { Person as SeoDataForArtist } from "v2/Components/Seo/Person"
+import { identity, pickBy } from "lodash"
+import React, { Component } from "react"
+import { Meta } from "react-head"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import { get } from "v2/Utils/get"
+import { Artist2MetaCanonicalLinkFragmentContainer as Artist2MetaCanonicalLink } from "./Artist2MetaCanonicalLink"
+
+interface Props {
+  artist: Artist2Meta_artist
+}
+
+// @ts-expect-error STRICT_NULL_CHECK
+type ArtworkNode = Artist2Meta_artist["artworks_connection"]["edges"][0]["node"]
```

Comment:
`non-blocking/comment:` I realize kind of typing is (and should be) frowned upon but was wondering if there's a solution for doing an existential access of an array like this in types. (I think this probably shakes out to an `any` because of that)

Reply:
(Yah this meta wa sall copy 🍝 -- stuff that seems crazy / old is likely crazy / old)